main.tf

```
@@ -36,6 +36,8 @@ resource "azurerm_subnet" "subnet" {
   address_prefixes                  = each.value["address_prefixes"]
   service_endpoints                 = each.value["service_endpoints"]
   private_endpoint_network_policies = "Enabled"
+  default_outbound_access_enabled   = each.value["default_outbound_access_enabled"] 
```

```suggestion
  default_outbound_access_enabled   = each.value["default_outbound_access_enabled"]  && startswith(var.vnet_name, "d-")
```

Kan vi starte med dette i tilfelle vi misser noe / har misforstått, så slipper vi å breake produksjon